src/virtual_secondary/CMakeLists.txt

```diff
@@ -0,0 +1,15 @@
+set(SOURCES managedsecondary.cc virtualsecondary.cc)
+
+set(HEADERS managedsecondary.h virtualsecondary.h)
+
+set(TARGET virtual-secondary)
```

Comment:
Small naming matter: we have the convention of using dashes for executables (kebab-case) and underscores for libraries (snake_case)

Reply:
ok